### ISSUE TITLE
Error: your wireless card  isn't supported 

### ISSUE THREAD

**Issue by h-gupta** [FluxionNetwork/fluxion#42]
### Do you want to request a *feature* or report a *bug*?
-> *Bug*
---

#### If the current behavior is a bug, please provide the steps to reproduce.
-> *A great way to do this is to provide screenshots and commands.*

It keeps appearing after selection to scan all channels
[2] Error: your wireless card  isn't supported  
Do you want exit? [Yes / No]:
--- 

#### Wireless adapter:
TP-LINK TL-WN722N

#### Does it support injection?
*Paste the output of `aireplay-ng -9`*

ioctl(SIOCSIWMODE) failed: Device or resource busy
14:27:53  Trying broadcast probe requests...
14:27:53  Injection is working!
14:27:55  Found 14 APs

#### Fluxion version:
*Paste the output of* `cat fluxion.sh | grep revision`

revision=10

#### Linux version:
*Paste the output of* `cat /proc/version` or `uname -r` 

4.9.0-kali4-amd64

I tested it on full kali 2017 also but it's not working.
It was working fine until a month ago.

**Comment by h-gupta** [comment]
Also monitor mode is also working fine .

**Comment by deltaxflux** [comment]
Recently, I change the interface function. It is maybe a problem with the latest version of airmon-ng.  To check if this is the case do following. 
```
su root
export FLUX_DEBUG=1
./fluxion
```
Another possibility is that you enable the interface and check with airmon-ng, in another terminal if the interface gets enabled. You should see something like: mon0 or wlan0mon depends on your airmon-ng version

**Comment by h-gupta** [comment]
Yeah , i think you are right.After enabling the debug mode i got this.
![flux](https://user-images.githubusercontent.com/19668711/28745433-0f058fae-7496-11e7-8b60-cffb343d9177.png)

**Comment by amitgiri723** [comment]
ESSID even BSSID not displaying in FLUXION ,(FLUXION SCANNER terminal)..

i use this command,  
#  export FLUXIONAirmonNG=1
#  ./fluxion.sh

but this command also didn't work.

please help!

Thank you!
